### PR TITLE
Added Dolmen as a frontend 

### PR DESCRIPTION
**Pull request by hra687261**
The purpose of this PR is to make it possible to use Dolmen as a front-end to Alt-Ergo and eventually make it the default front-end instead of the current one. 

To use Dolmen as a frontend: 
```./alt-ergo --frontend dolmen```

### CODE REVIEW

File: src/lib/frontend/d_cnf.ml
Comment (Gbury) [inline src/lib/frontend/d_cnf.ml]: you should at the very least, assert that the absolute path is empty, or else that will cause problems later

Comment (hra687261) [reply src/lib/frontend/d_cnf.ml]: Done.

**Comment by hra687261** [comment]
This PR will have to wait for a new release of dolmen before being reviewed/merged as it uses its latest version

**Review by Halbaroth — APPROVED** [review]

**Pull request merged**
